docs/excel/custom-functions-save-state.md

```diff
@@ -0,0 +1,51 @@
+---
+ms.date: 04/25/2019
+description: Use `Office.Storage` to save state with custom functions. 
+title: Save and share state in custom functions (preview)
+localization_priority: Priority
+---
+
+## Save and share state in custom functions
+
+The main storage location for custom functions is `Office.Storage`. Storage is limited to 10 MB per domain (which may be shared across multiple add-ins). On Excel for Windows, `Office.Storage` is a separate location within the custom functions runtime, but for Excel Online and Excel for Mac, `Office.Storage` is the same as the browser's `localStorage`. `Office.Storage` is a useful storage location that can be accessed by both custom functions and your add-in's task pane.
+
+There are multiple ways to use `Office.Storage` to save and share state:
```

Comment:
Consider "There are multiple ways to use `Office.Storage` to save and share state:" -> "There are multiple ways to use `Office.Storage` for state management:"